### PR TITLE
Style for Payment, Payment Success and Order History

### PR DESCRIPTION
**Pull request by Djmiller1717**
- Updated the Nav Bar so that the Order History only shows when a user is logged in.
- Added another Disney costume to the seed file.
- Styled the Payment screen
- Styled the Payment Success screen
- Styled the Order History screen

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ const costumes = [
     discount: discounts[Math.floor(Math.random() * (discounts.length - 1))],
     madeIn: faker.address.country(),
   },
+  {
+    costumeName: 'Tarzan',
+    price: 33.43,
+    categortyId: 2,
+    imageUrl: '/costumeImages/disney/tarzan.jpeg',
+  },
```
Comment (fugoto) [inline seed.js]: can you add the discount, material, madeIn seeding (you can copy paste from another costume)

File: client/components/OrderHistory.js [hunk 2]
```diff
@@ -14,28 +14,31 @@ class OrderHistory extends Component {
...
               const readableDate = date.toDateString();
               return (
-                <li key={order.id}>
-                  <div>{readableDate}</div>
-                  <div>
-                    Total: $
+                <div key={order.id}>
+                  <div className="orderText">Order Date: {readableDate}</div>
+                  <div className="orderText">
+                    Item Cost: $
```
Comment (fugoto) [inline client/components/OrderHistory.js]: minor point but can you call this Total: "instead of Item Cost"?

File: seed.js (hunk 1 above)
Comment (Djmiller1717) [reply seed.js]: updated!

File: client/components/OrderHistory.js (hunk 2 above)
Comment (Djmiller1717) [reply client/components/OrderHistory.js]: updated!

**Review by fugoto — APPROVED** [review]
yay!

**Pull request merged**